Source/Csla/Server/DataPortal.cs

```diff
@@ -774,7 +775,7 @@ private void ClearContext(DataPortalContext context)
       if (!context.IsRemotePortal) return;
       _applicationContext.Clear();
       if (SecurityOptions.FlowSecurityPrincipalFromClient)
-        _applicationContext.User = null;
+        _applicationContext.User = new ClaimsPrincipal(new ClaimsIdentity());
```

Comment:
You can just create the new principal - it defaults to creating a new identity if you don't provide one.

Reply:
Now obsolete. I set it to `null` again.

Reply:
That's not true. Using the default ctor the principal does not have an identity associated with it.
See https://github.com/dotnet/runtime/blob/1d1bf92fcf43aa6981804dc53c5174445069c9e4/src/libraries/System.Security.Claims/src/System/Security/Claims/ClaimsPrincipal.cs#L99-L104